migrations/20170126145822_replace_document_type_with_document_categories.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const _ = require('lodash');
+const Promise = require('bluebird');
+
+const typeToCategoryMapping = {
+  blank_consent_form: 33,
```

Comment:
@nightsh I figured the `name` and `group` would probably change more often than the ID, wouldn't you agree?